### PR TITLE
GSG: Added GPRC examples on ginkgo testing

### PR DESCRIPTION
**Pull request by eloycoto**

### CODE REVIEW

File: test/k8sT/Nightly.go [hunk 1]
```diff
@@ -440,4 +440,77 @@ var _ = Describe("NightlyExamples", func() {
 			res.ExpectSuccess("Cannot curl service after update")
 		})
 	})
+
+	Context("Guetting started guides", func() {
```
Comment (ianvernon) [inline test/k8sT/Nightly.go]: Typo: Guetting --> Getting

File: test/k8sT/Nightly.go [hunk 2]
```diff
@@ -440,4 +440,77 @@ var _ = Describe("NightlyExamples", func() {
...
+
+		AfterEach(func() {
+			err := kubectl.WaitCleanAllTerminatingPods()
+			Expect(err).To(BeNil(), "cannot clean all terminating pods")
+		})
+
+		It("GRPC example", func() {
+
+			AppManifest := helpers.GetFilePath(
+				"../examples/kubernetes-grpc/cc-door-app.yaml")
```
Comment (ianvernon) [inline test/k8sT/Nightly.go]: Avoid hardcoding of common paths and put them into variables.

**Review by ianvernon — CHANGES_REQUESTED** [review]

**Comment by eloycoto** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

**Comment by ianvernon** [comment]
test-nightly

**Comment by eloycoto** [comment]
@ianvernon this test worked but failed due other problems in nightly. I think that can be merged.

**Pull request merged**